ProcessLib/TwoPhaseFlowWithPP/TwoPhaseFlowWithPPLocalAssembler.h

```diff
@@ -114,13 +143,10 @@ class TwoPhaseFlowWithPPLocalAssembler
     std::vector<ShapeMatrices> _shape_matrices;
 
     TwoPhaseFlowWithPPProcessData const& _process_data;
+    std::vector<IntegrationPointData<NodalMatrixType>> _ip_data;
 
-    // Note: currently only isothermal case is considered, so the temperature is
-    // assumed to be const
-    // the variation of temperature will be taken into account in future
-    double _temperature = 293.15;
     std::vector<double> _saturation;
-    std::vector<double> _pressure_wetting;
+    std::vector<double> _pressure_wet;
```

Comment:
Can `` _pressure_wet`` and ``_saturation`` be set local in the function where they are used? They may cause thread safe problem (e.g. data race) if openmpi is applied for local/global assembly.